crates/driver/src/tests/setup/solver.rs

```diff
@@ -133,6 +133,7 @@ impl Solver {
                 "buyAmount": buy_amount,
                 "fullBuyAmount": if config.quote { buy_amount } else { quote.buy_amount().to_string() },
                 "validTo": quote.order.valid_to,
+                "created": quote.order.created,
```

Comment:
This field is going to be always 0 because you are not assigning anything to `quote.order.created` before this. I believe this is the the root cause of the test failures.

Reply:
> I believe this is the the root cause of the test failures.

Here I was referring to the [test-driver test failures](https://github.com/cowprotocol/services/actions/runs/15665833560/job/44315015120?pr=3435#step:8:267)